app/src/main/java/com/example/ubclaunchpad/launchnote/photoBrowser/PhotoBrowserActivity.kt

```diff
@@ -61,8 +73,40 @@ class PhotoBrowserActivity : BaseActivity() {
             view_pager.currentItem = ALL_FRAGMENT
             updateButtons(ALL_FRAGMENT)
         }
+
+        // Restore state of fragments
+        (0 until NUMBER_OF_FRAGMENTS)
```

Comment:
you could probably use [getFragments()](https://developer.android.com/reference/android/support/v4/app/FragmentManager.html#getFragments()) instead

Reply:
hmm, dont think so getFragments -. "Get a list of all fragments that are currently added to the FragmentManager." but we dont want all displayed fragments, just the ones inside of the view pager. Unfortunately there is no way of knowing